message_object.py

```diff
@@ -116,6 +116,10 @@ def __init__(self, **kwargs):
     self.x_velocity = kwargs.get("x_velocity")
     self.y_velocity = kwargs.get("y_velocity")
 
+    # Data from the Nav system with our target velocity. (m/s)
+    self.x_target_velocity = kwargs.get("x_target_velocity")
```

Comment:
Getting the current velocity is easy, but getting the "desired" velocity will be hard if not impossible. What is the need for this information?